devops/charts/server/values_test.yaml

```diff
@@ -28,10 +28,10 @@ env:
 
 resources:
   requests:
-    memory: 92Mi
+    memory: 184Mi
     cpu: 10m
   limits:
-    memory: 128Mi
+    memory: 256Mi
```

Comment:
Generally you don't want `limits.memory` to be much higher than `request.memory`. The reason is, if your Pod is running happily over the `request.memory` and a node becomes under memory pressure it will evict (forcefully move) your pod to another node. CPU does not work this way which is why it's generally Ok to have request 10m and limit 100m. 

This isn't wrong, just be aware of how this works. Since this is JS and JS uses garbage collection it may be just fine because it will periodically reduce the memory footprint. 

Generally, if your pod mostly runs at 164MB then give it a request/limit of 192. That is what it needs + a bit of wiggle room. Set request = limit. If you're getting OOMKilled it may indicate a memory leak.

Reply:
adjusted